client/src/Utils/api.js

```diff
@@ -55,4 +78,25 @@ const postCode = async (code, redirectUrl) => {
   }
 };
 
-export { checkForGmailToken, getAuthUrl, postCode };
+const getUser = async (userId, token) => {
```

Comment:
Instead of adding the token into each API call, we can use axios global headers to always add it. We just need to remember to remove it if someone exits. 

You should probably put the logout function also in one of the HOCs - that way you can control that logic there as well

Reply:
Thanks for the tip. Have set the axios global header in withAuthentication and removed it on logout. Have moved the logout function to context too.

One small problem was the axios global header seemed to reset during the Gmail authorisation process, presumably because we left our domain. For the axios api call after returning to our site from Google, I reassigned the token to axios global headers.

Reply:
Actually, had to move it to api.js as having it in withAuthentication was causing unpredictable behavior.